airflow/providers/smtp/operators/smtp.py

```diff
@@ -80,9 +80,9 @@ def execute(self, context: Context):
         with SmtpHook(smtp_conn_id=self.conn_id) as smtp_hook:
             return smtp_hook.send_email_smtp(
                 self.to,
-                self.from_email,
                 self.subject,
                 self.html_content,
+                self.from_email,
```

Comment:
I prefer to provide the name of each argument to avoid the similar problems in the future, wdyt?
```diff
@@ -79,10 +79,10 @@ class EmailOperator(BaseOperator):
     def execute(self, context: Context):
         with SmtpHook(smtp_conn_id=self.conn_id) as smtp_hook:
             return smtp_hook.send_email_smtp(
-                self.to,
-                self.from_email,
-                self.subject,
-                self.html_content,
+                to=self.to,
+                subject=self.subject,
+                html_content=self.html_content,
+                from_email=self.from_email,
                 files=self.files,
                 cc=self.cc,
                 bcc=self.bcc,
```